models/blur/ethereum/blur_ethereum_events.sql

```diff
@@ -134,4 +134,74 @@ LEFT JOIN {{ ref('nft_ethereum_transfers') }} seller_fix ON seller_fix.block_tim
 {% if is_incremental() %}
 WHERE bm.evt_block_time >= date_trunc("day", now() - interval '1 week')
 {% endif %}
+
+UNION ALL
+
+SELECT 'ethereum' AS blockchain
+, 'blur' AS project
+, 'v1' AS version
+, date_trunc('day', s.evt_block_time) AS block_date
+, s.evt_block_time AS block_time
+, s.evt_block_number AS block_number
+, get_json_object(s.offer[0], '$.identifier') AS token_id
+, tr.token_standard AS token_standard
+, nft_tok.name AS collection
+, CASE WHEN get_json_object(s.offer[0], '$.amount')=1 THEN 'Single Item Trade' ELSE 'Bundle Trade' END AS trade_type
+, get_json_object(s.offer[0], '$.amount') AS number_of_items
```

Comment:
minor point, but can you cast this column (and all others that do so in above query) to the same datatypes? while spark will infer correctly based on above query, it's more for consistency/readability. this will apply to a few columns i'm sure, using this one as example to comment